docs/osd.md

```diff
@@ -0,0 +1,165 @@
+| ![056.png](osd_images/056.png) | 0x38 | 056 |                                 |       |
+| ![057.png](osd_images/057.png) | 0x39 | 057 |                                 |       |
+| ![058.png](osd_images/058.png) | 0x3A | 058 |                                 |       |
+| ![059.png](osd_images/059.png) | 0x3B | 059 |                                 |       |
+| ![060.png](osd_images/060.png) | 0x3C | 060 |                                 |       |
+| ![061.png](osd_images/061.png) | 0x3D | 061 |                                 |       |
+| ![062.png](osd_images/062.png) | 0x3E | 062 |                                 |       |
+| ![063.png](osd_images/063.png) | 0x3F | 063 |                                 |       |
+| ![064.png](osd_images/064.png) | 0x40 | 064 |                                 |       |
+| ![065.png](osd_images/065.png) | 0x41 | 065 |                                 |       |
+| ![066.png](osd_images/066.png) | 0x42 | 066 |                                 |       |
+| ![067.png](osd_images/067.png) | 0x43 | 067 |                                 |       |
+| ![068.png](osd_images/068.png) | 0x44 | 068 |                                 |       |
+| ![069.png](osd_images/069.png) | 0x45 | 069 |                                 |       |
+| ![070.png](osd_images/070.png) | 0x46 | 070 |                                 |       |
+| ![071.png](osd_images/071.png) | 0x47 | 071 |                                 |       |
+| ![072.png](osd_images/072.png) | 0x48 | 072 |                                 |       |
+| ![073.png](osd_images/073.png) | 0x49 | 073 |                                 |       |
+| ![074.png](osd_images/074.png) | 0x4A | 074 |                                 |       |
+| ![075.png](osd_images/075.png) | 0x4B | 075 |                                 |       |
+| ![076.png](osd_images/076.png) | 0x4C | 076 |                                 |       |
+| ![077.png](osd_images/077.png) | 0x4D | 077 |                                 |       |
+| ![078.png](osd_images/078.png) | 0x4E | 078 |                                 |       |
+| ![079.png](osd_images/079.png) | 0x4F | 079 |                                 |       |
+| ![080.png](osd_images/080.png) | 0x50 | 080 |                                 |       |
+| ![081.png](osd_images/081.png) | 0x51 | 081 |                                 |       |
+| ![082.png](osd_images/082.png) | 0x52 | 082 |                                 |       |
+| ![083.png](osd_images/083.png) | 0x53 | 083 |                                 |       |
+| ![084.png](osd_images/084.png) | 0x54 | 084 |                                 |       |
+| ![085.png](osd_images/085.png) | 0x55 | 085 |                                 |       |
+| ![086.png](osd_images/086.png) | 0x56 | 086 |                                 |       |
+| ![087.png](osd_images/087.png) | 0x57 | 087 | SYM_WATT                        | Used in OSD.c      |
```

Comment:
lol. Nice that somebody bothered to define a symbol for `W`. :laughing:

Reply:
@mikeller it would appear that the original fonts had seperate symbols for Watts and Volts (i.e. not `W` and `V`) - The Watt glyph has been replaced by `W` but we still have a seperate glyphs for `V` and Volts (`0x56` vs `0x06`)

Reply:
I assume so - I just find it funny.

Reply:
I agree but I find it slighty more funny that we need a special character for Volts as a normal `V` obviously isn't good enough!

Reply:
Good point, @frozenskys.